.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Sentinel CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [8, 11]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+          architecture: x64
+
+      - name: Build with Maven
+        run: mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V -DminimumPriority=1
```

Comment:
Here we'll need to execute both build and test.

FYI (config in Travis CI): https://docs.travis-ci.com/user/languages/java/#maven-dependency-management

Reply:
OK, I will change it later.